lib/spec/swagger/utils.js

```diff
@@ -266,9 +266,28 @@ function prepareSwaggerMethod (schema, ref, swaggerObject) {
   return swaggerMethod
 }
 
+function transformDefs (jsonSchema) {
+  if (typeof jsonSchema === 'object' && jsonSchema !== null) {
+    Object.keys(jsonSchema).forEach(function (key) {
+      if (key === '$ref') {
+        jsonSchema[key] = `#/definitions/${jsonSchema[key]}`
+      } else if (key === 'examples' && Array.isArray(jsonSchema[key]) && (jsonSchema[key].length > 1)) {
+        jsonSchema.examples = convertExamplesArrayToObject(jsonSchema.examples)
```

Comment:
Is this case tested?

Reply:
not at the moment, I'm not sure why the code is handing `examples`.  Should it be tested, or removed?

Reply:
We landed the https://github.com/fastify/fastify-swagger/pull/472 PR

is this needed?

Reply:
I'd say, this PR is still needed as this PR takes care of both Swagger and OpenAPI,  that otehr PR only takes care of OpenAPI?

Does another test need to be added for when the user does not tweak `buildLocalReference`, or is returning the $id now preferred?

Reply:
closed in favour of https://github.com/fastify/fastify-swagger/pull/479